### PR TITLE
feat: Suport CREATE POLICY (for psqldef)

### PR DESCRIPTION
**Pull request by jumpeiMano**
I have made additional implementations to support the following features:
- [POLICY](https://www.postgresql.org/docs/current/sql-createpolicy.html) (for psqldef)

And I also fix a bug of psqldef. When I used `ALTER TABLE ONLY` statements to set PRIMARY KEY, I received some errors like this.
```
unexpected ddl type in generateDDLs: &{ALTER TABLE ONLY dbname.tablename
    ADD CONSTRAINT pkname PRIMARY KEY (columnname) dbname.tablename {pkname  [{columnname <nil>}] false false }}
exit status 1
```
So I fixed it.

### CODE REVIEW

**Comment by k0kubun** [comment]
Great work :smile: 

Could you make the CI successful first though? Also, if possible, it'd be nice if you split this PR into three, VIEW support, POLICY support, and others if any. Just nesting PRs is fine. I believe this will help your investigation of the CI failure as well.

**Comment by jumpeiMano** [comment]
@k0kubun Thanks! At first I made https://github.com/k0kubun/sqldef/pull/78 from this PR.

**Comment by k0kubun** [comment]
I'm sorry, but your original PR ended up conflicting with the split PR. Could you rebase this PR from the current master?

**Comment by jumpeiMano** [comment]
@k0kubun Now I merged current master branch and fix some bugs.

File: schema/ast.go [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package schema
 
+import "github.com/k0kubun/sqldef/sqlparser"
```
Comment (k0kubun) [inline schema/ast.go:3]: The whole point of `github.com/k0kubun/sqldef/schema` package is to decouple `sqlparser` from other sqldef implementations. Thus this file should not import `sqlparser`. Could you consider not to use `sqlparser.Permissive` in this package?

**Comment by k0kubun** [comment]
Please don't add `.travis.yml`.

**Comment by jumpeiMano** [comment]
> Please don't add `.travis.yml`.

Sorry, I deleted it.

File: schema/ast.go (hunk 1 above)
Comment (jumpeiMano) [reply schema/ast.go]: OK, I fixed it.
https://github.com/k0kubun/sqldef/pull/77/commits/a2e003ba5005cc02d9f317847a4b76a4310a4978

**Review by k0kubun — APPROVED** [review]
Great job :clap:

**Pull request merged**